deployment/prometheus/prometheus.yml

```diff
@@ -7,4 +7,3 @@ scrape_configs:
     static_configs:
```

Comment:
You should replace this file with your `config.yml` and then you won't need to build a new container.